css/properties/-webkit-line-align.json

```diff
@@ -5,7 +5,8 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "19",
+              "version_removed": "35"
```

Comment:
✅ Added in Chrome 19 (https://github.com/caugner/css-properties/commit/218727e3046d6dbfd51e140256e4d830c5de45ea).
:x: Removed in Chrome 34 (https://github.com/caugner/css-properties/commit/4560f7fda4a8b2300fc5c9135085a3ed7b7651ee).

```suggestion
              "version_added": "19",
              "version_removed": "34"
```